sigtracker/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+from eveonline.models import EveCharacter
+from eveonline.models import EveCorporationInfo
+
+
+class sigtracker(models.Model):
```

Comment:
That would be great for auditing. I'd only display the "created by" field to those with the signature management permission personally.

The created time should be visible on the webpage.